app/src/main/java/io/gnosis/safe/ui/safe/send_funds/AddOwnerFirstFragment.kt

```diff
@@ -0,0 +1,44 @@
+package io.gnosis.safe.ui.safe.send_funds
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
+import io.gnosis.safe.R
+import io.gnosis.safe.ScreenId
+import io.gnosis.safe.databinding.FragmentAddOwnerFirstBinding
+import io.gnosis.safe.di.components.ViewComponent
+import io.gnosis.safe.ui.base.fragment.BaseViewBindingFragment
+import java.math.BigInteger
+
+class AddOwnerFirstFragment : BaseViewBindingFragment<FragmentAddOwnerFirstBinding>() {
```

Comment:
Do we need to track when AddOwnerFirstFragment was opened? Maybe not a question for this PR. But where would I put it?

Reply:
it is being tracked already. This fragment has screen id `screen_add_owner_to_send_assets`